animation_nodes/nodes/mesh/mesh_object_input.py

```diff
@@ -195,3 +213,32 @@ def loadCustomAttributes(self, mesh, sourceMesh, object):
                                                      object.data.an.getCustomAttribute(customAttributeName)))
         else:
             self.setErrorMessage("Object is in edit mode.")
+
+    def execute_All_WithoutModifiers(self, object, vertexGroup):
+        vertexAmount = len(object.data.vertices)
+        weights = DoubleList(length = vertexAmount)
+        getWeight = vertexGroup.weight
+
+        for i in range(vertexAmount):
+            try: weights[i] = getWeight(i)
+            except: weights[i] = 0
+        return weights
+
+    def execute_All_WithModifiers(self, object, vertexGroup, scene):
```

Comment:
Just remove this for now. Seems like a Blender limitation.

Reply:
But how we are going to handle the data length? Or should we just add zero weight?

Reply:
For now return zero weights. We can look into this case in more details later.